### PR TITLE
feat: add telegram notifier

### PR DESCRIPTION
**Pull request by cehali**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,3 +11,7 @@ or
 ## Running
 
 `node ./src/run-monitoring-service.js`
+
+## Telegram integration
+
+In order to integrate the monitoring notifier with telegram chat or channel, you need to populate `TELEGRAM_TOKEN` and `TELEGRAM_CHAT_ID` in the environment variables file. To get those variables you need to create a bot using [BotFather](#https://core.telegram.org/bots#3-how-do-i-create-a-bot) and add it to your chat or channel. After creating the bot you will receive an `access token` which you should use as `TELEGRAM_TOKEN`. Chat id you can be found from bot updates response. It can be fetched from URL `https://api.telegram.org/bot<YourBOTToken>/getUpdates`. The response should be an object with `chat` attribute and `id` inside it.
```
Comment (hatskier) [inline README.md:17]: `Chat id you can be found` -> `Chat id can be found`

File: src/helpers/db-connector.ts [hunk 2]
```diff
@@ -5,3 +5,8 @@ export const connectToRemoteMongo = async () => {
   await mongoose.connect(mongoDbUrl);
   console.log("Connected to MongoDB");
 };
+
+export const disconnectFromRemoteMongo = async () => {
```
Comment (hatskier) [inline src/helpers/db-connector.ts:9]: Why do we need to disconnect from DB?

**Review by hatskier — CHANGES_REQUESTED** [review]
Let's change telegram report to smth like "Errors: 12, Warnings: 100". Later we can also add a link to more detailed report with UI.
Also, can you enable sending only critical notifications on telegram - only in case when there are errors (not only warnings)?

File: src/helpers/db-connector.ts (hunk 2 above)
Comment (cehali) [reply src/helpers/db-connector.ts:9]: I used it as a separate script, I will delete connect and disconnect to DB

**Review by hatskier — APPROVED** [review]

**Pull request merged**